docs/LearningOutcomes.adoc

```diff
@@ -27,7 +27,7 @@ Note how the <<DeveloperGuide#architecture, Developer Guide>> describes the high
 
```

Comment:
The link to the repoURL should still be `seedu/address/...` . All the other links will break, otherwise.